Source/SwiftLintFramework/Rules/NoFallthroughRuleOnlyRule.swift

```diff
@@ -0,0 +1,109 @@
+import Foundation
+import SourceKittenFramework
+
+public struct NoFallthroughOnlyRule: ASTRule, OptInRule, ConfigurationProviderRule {
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "no_fallthrough_only",
+        name: "No Fallthrough Only",
+        description: "Fallthroughs can only be used if the `case` contains at least one other statement.",
+        kind: .idiomatic,
+        nonTriggeringExamples: [
```

Comment:
It'd be nice to have more complex test cases here, such as using pattern matching, or `default` or `break` statements.